cpp/ql/test/query-tests/Likely Bugs/Underspecified Functions/MistypedFunctionArguments.expected

```diff
@@ -18,3 +18,4 @@
 | test.c:58:3:58:24 | call to defined_with_long_long | Calling $@: argument $@ of type $@ is incompatible with parameter $@. | test.c:103:11:103:32 | defined_with_long_long | defined_with_long_long | test.c:58:26:58:26 | 3 | 3 | file://:0:0:0:0 | int | int | test.c:103:44:103:45 | ll | long long ll |
 | test.c:60:3:60:21 | call to defined_with_double | Calling $@: argument $@ of type $@ is incompatible with parameter $@. | test.c:99:8:99:26 | defined_with_double | defined_with_double | test.c:60:23:60:25 | 2 | 2 | file://:0:0:0:0 | long long | long long | test.c:99:35:99:35 | d | double d |
 | test.c:61:3:61:24 | call to defined_with_long_long | Calling $@: argument $@ of type $@ is incompatible with parameter $@. | test.c:103:11:103:32 | defined_with_long_long | defined_with_long_long | test.c:61:26:61:31 | 3500000000000000.0 | 3500000000000000.0 | file://:0:0:0:0 | double | double | test.c:103:44:103:45 | ll | long long ll |
+| test.c:137:3:137:27 | call to implicit_declaration_good | Calling $@: argument $@ of type $@ is incompatible with parameter $@. | test2.c:8:6:8:30 | implicit_declaration_good | implicit_declaration_good | test.c:137:35:137:38 | 2.0 | 2.0 | file://:0:0:0:0 | float | float | test2.c:8:52:8:52 | f | float f |
```

Comment:
Do we know what's going on with this result?

Reply:
Yes. The last expression gets converted into a `double` causing `MistypedFunctionArguments.ql` to flag it is a mismatch (as it's expecting a `float`). It's the same kind of test as this one (at least I think that's what it's meant to catch): https://github.com/Semmle/ql/blob/1bc3829a72a14f28f454a89ff9459c73cd019c9c/cpp/ql/test/query-tests/Likely%20Bugs/Underspecified%20Functions/test.c#L51
(except that the definition is in a different file for this new test)

Reply:
OK.  I don't think it's a useful result, but it's nothing to do with this PR.

Reply:
I've removed the testcase now.